preset/preset_builtin.list

```diff
@@ -1,6 +1,6 @@
 <resources>
     <section name="PresetTemplate">
-        <integer name="VersionMajor" value="27" />
+        <integer name="VersionMajor" value="29" />
```

Comment:
You bumped the version number twice.  Not really necessary if these are going in together.